frontend/src/concepts/pipelines/content/tables/experiment/ExperimentTableRow.tsx

```diff
@@ -30,6 +30,7 @@ const ExperimentTableRow: React.FC<ExperimentTableRowProps> = ({
       <CheckboxTd id={experiment.experiment_id} isChecked={isChecked} onToggle={onToggleCheck} />
       <Td dataLabel="Experiment">
         <TableRowTitleDescription
+          boldTitle={false}
```

Comment:
The default is `false`
```suggestion
```

Reply:
Ah yep, the update to the default was done after this update and I forgot to remove this. Will go back through to clean up this prop usage.